README.md

```diff
@@ -105,15 +105,15 @@ Let's take the example of ```snails-backend-recentf``` plugin:
 (provide 'snails-backend-recentf)
 ```
 
-* :name parameter is the name of your plugin, you must use a unqie name. Snails distinguishes the results of different plugins based on the plugin name.
+* :name parameter is the name of your plugin, and it must be unique. Snails distinguishes the results of different plugins based on the plugin name.
 
-* :candidate-filter is fitler function, ```input``` is user input content, you need return candidate list to snails framework, element of candidate list is format as ```(list display-name candidate-content)```, first of candidate element ```display-name``` is the string presented to the user, second of candidate elemenet ```candidate-content``` is the string pass to ```candidate-do``` callback at below. If nothing search, please renturn nil, snails will hide the backend result.
+* :candidate-filter is the filter function. ```input``` is user input content, and you need to return a candidate list to the snails framework, where elements of the candidate list are formatted as ```(list display-name candidate-content)```. The first candidate element ```display-name``` is the string presented to the user, and the second candidate element ```candidate-content``` is the string passed to the ```candidate-do``` callback below. If nothing was found, please return nil, and snails will hide the backend result.
```

Comment:
Where it previously said "If nothing search" I interpreted to mean "If nothing was found" - did you instead mean if nothing was searched for?